src/api/code-challenge/content-types/code-challenge/general.ts

```diff
@@ -0,0 +1,50 @@
+// andy - Not Directly Used in MetaTests
```

Comment:
I'm a bit confused. Why is this file added if it isn't used in metaTests? Is it just apart of the general migration to TS?

Reply:
I would remove the comment then. Also, can you remove any unused functions and commented code?

Reply:
Done